homeassistant/components/ohme/sensor.py

```diff
@@ -61,6 +67,31 @@ class OhmeSensorDescription(OhmeEntityDescription, SensorEntityDescription):
         state_class=SensorStateClass.TOTAL_INCREASING,
         value_fn=lambda client: client.energy,
     ),
+    OhmeSensorDescription(
+        key="battery",
+        native_unit_of_measurement=PERCENTAGE,
+        device_class=SensorDeviceClass.BATTERY,
+        suggested_display_precision=0,
+        value_fn=lambda client: client.battery,
+    ),
+    OhmeSensorDescription(
+        key="slot_list",
+        translation_key="slot_list",
+        value_fn=lambda client: ", ".join(str(x) for x in client.slots)
+        or STATE_UNKNOWN,
+    ),
```

Comment:
What's a slot list?

Reply:
I guess doing the doc PR in advance would've been useful here!

When you plug your car in, the car doesn't charge, rather the charger gets a list of time slots from your energy provider.

(Basically you just set your target percentage and what time you need the car charged by - I'll add these controls in future PRs!)

Reply:
And this sensor just shows a string for the slots you've been assigned like: "00:15-03:30, 04:00-05:00"

Reply:
As these are a list of values, I'd rather expect them as a service response (or maybe even as a calendar entity, but I don't believe this is very useful in this chase)

Reply:
As in update the sensor based on a service response or just have a service that provides the list and no sensors?

Reply:
The latter

Reply:
Something more like that...